.drone.yml

```diff
@@ -44,29 +57,54 @@ steps:
 #    - push
 #    - pull_request
 
-- name: Push Image to Quay.io
-  pull: if-not-exists
-  image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind:latest
+- name: maintenance-image-scan
+  image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/anchore-submission:latest
+  pull: always
+  failure: ignore
   environment:
-    DOCKER_PASSWORD:
-      from_secret: DOCKER_PASSWORD
-    DOCKER_USERNAME:
-      from_secret: DOCKER_USERNAME
-    DOCKER_REGISTRY_URL:
-      from_secret: DOCKER_REGISTRY_URL
-  commands:
-  - docker login -u="$${DOCKER_USERNAME}" -p=$${DOCKER_PASSWORD} $${DOCKER_REGISTRY_URL}
-  - docker tag egar_ui-public-site:$${DRONE_COMMIT} $${DOCKER_REGISTRY_URL}/ukhomeofficedigital/egar-public-site-ui:$${DRONE_COMMIT}
-  - docker push quay.io/ukhomeofficedigital/egar-public-site-ui:$${DRONE_COMMIT}
+    IMAGE_NAME: quay.io/ukhomeofficedigital/egar-public-site-ui:${DRONE_COMMIT}
+    TOLERATE: high
   when:
-    branch:
-    - master
-    - hotfix/*
     event:
+    - pull_request
     - push
```

Comment:
Do we need to run the scan for every commit ? Perhaps after a PR is created is sufficient? Happy to know the reason behind this.